src/wxflow/task.py

```diff
@@ -91,3 +96,17 @@ def clean(self):
         Methods to clean after execution and finalization prior to closing out a task
         """
         pass
+
+    def extend_task_config(self, local_dict: Union[Dict, AttrDict]) -> None:
+        """
+        Extend task_config attribute-dictionary with another dictionary
+        """
+
+        self.task_config = AttrDict(**self.task_config, **local_dict)
+
+    def stage_files(self, path: str) -> None:
+        """
+        Pass dictionary, created by parsing Jinja2-templated YAML, to file handler
+        """
+
+        FileHandler(parse_j2yaml(path, self.task_config)).sync()
```

Comment:
I don't think this belongs here in this form.
If we want a `stage` method in the task, its input should be the a fully formed list and not something that needs to be rendered.

Reply:
That's basically what FileHandler does. Theres so many places where we parse a jinja2-templated yaml and then pass to the filehandler, I figured it would be useful to make a class method out of it. I can remove it though.

Reply:
Yes. But, if there is an issue in the parsing, and one needs to print the resolved yaml, one will have to come in `wxflow` and do debugging.
Besides, passing only the necessary keys to the yaml path, rather than the entire task_config.  would help in debugging.